src/pages/Home/Home.page.jsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import NavBar from '../../components/Navbar';
 import VideoCardList from '../../components/VideoCardList';
+import useGapi from '../../hooks/useGapi';
+import { useParams } from 'react-router-dom';
 
 function HomePage() {
+  
+  const { searchQuery } = useParams();
+  let { videos } = useGapi( searchQuery ? searchQuery : "", null)
```

Comment:
If you're not changing the videos' variable, I recommend using `const` so we keep closer to a pure function

Reply:
Perfect, i was actually doubting about which reserved word to use, thank you for clarifying it.